webhook/configmaps/configmaps.go

```diff
@@ -137,9 +137,15 @@ func (ac *reconciler) reconcileValidatingWebhook(ctx context.Context, caCert []b
 
 	webhook := configuredWebhook.DeepCopy()
 
-	// Clear out any previous (bad) OwnerReferences.
-	// See: https://github.com/knative/serving/issues/5845
-	webhook.OwnerReferences = nil
+	// Set the owner to namespace.
+	if len(webhook.OwnerReferences) == 0 {
```

Comment:
Should we fully reconcile this? I.e. make sure it's always the correct ownerRef, even if this is non-zero (i.e. after the first reconcilation?)